setup.py

```diff
@@ -73,19 +73,21 @@ def build_extension(self, ext):
 
 setup(
     name='nestpy',
-    version='1.5.2',
+    version='1.5.3',
     author='Sophia Farrell',
     author_email='sja5@rice.edu',
     description='Python bindings for the NEST noble element simulations',
     long_description=readme + '\n\n' + history,
     long_description_content_type="text/markdown",
-    packages=find_packages('src'),
+    # Include lib such that recompilation under e.g. different numpy versions works
+    packages=find_packages(include=['src', 'lib']),
```

Comment:
So `'lib'` includes pybind11, not 100% sure how this is necessary when considering numpy versions... though I do agree the pybind would be useful to keep if you are going to change and recompile `nestpy`, for most people who are changing like that, we'd aim that they install from source and not pip... 
However, if there is a reason why we need to keep pybind11 that I'm not understanding, that's fine, but otherwise it is just baggage

Reply:
Can be solved at a later date

Reply:
Actually the way this is written is why the packages for our test installations are empty, I've found. I'm going to leave it as is and then we can adjust this later to resolve how to include pybind11 in a next patch (but a new patch is urgent.)